package.json

```diff
@@ -37,11 +38,12 @@
     "@sentry/cli-linux-arm64": "2.53.0",
     "@sentry/cli-linux-i686": "2.53.0",
     "@sentry/cli-linux-x64": "2.53.0",
+    "@sentry/cli-win32-arm64": "2.53.0",
     "@sentry/cli-win32-i686": "2.53.0",
-    "@sentry/cli-win32-x64": "2.53.0",
-    "@sentry/cli-win32-arm64": "2.53.0"
+    "@sentry/cli-win32-x64": "2.53.0"
   },
   "scripts": {
+    "preinstall": "npx npm-force-resolutions",
```

Comment:
Is this necessary for end users? `preinstall` (just like `postinstall`) runs when users install the package: https://docs.npmjs.com/cli/v8/commands/npm-install

Reply:
I am following these instructions: https://www.npmjs.com/package/npm-force-resolutions.

Unfortunately, `npm` version 6 (used in Node Test 10.x) is not compatible with the `"overrides"` section, so we need a package like `npm-force-resolutions` to pin a specific version of `browserslist`.